ui/packages/consul-ui/tests/steps/assertions/page.js

```diff
@@ -41,6 +41,9 @@ export default function(scenario, assert, find, currentPage, $) {
       assert[negative ? 'notOk' : 'ok'](res, this.step);
       setTimeout(() => next());
     })
+    .then('I wait a tick', function() {
+      return new Promise(resolve => setTimeout(resolve, 0));
+    })
```

Comment:
See above comment on not needing this. But also FYI for any future step additions, this isn't asserting anything, and isn't concerned with page objects (the `page` name of the file), so ideally it wouldn't be in this folder/file.

We have organised the steps folder into `assertions`, `interactions`, `debug` and `doubles` (mocking/setting up), so steps are easier to find based on those categories (and filenames within those would also help), and I think we at least mostly have kept to that. Overall, nothing to do here (apart from remove this) so this is just for knowledge sharing for future.

Reply:
thanks for the background info